packages/form/src/module.ts

```diff
@@ -37,7 +37,7 @@ export type FieldState = {
 
 export type FormState<TKey extends string | number | symbol = string> = {
 	isSubmitPending: boolean;
-	submitStatus: "idle" | "validating" | "submitting";
+	submitStatus: "idle" | "validating" | "submitting" | "submitted";
```

Comment:
```suggestion
	submitStatus: "idle" | "validating" | "submitting";
```

For consistency with the Astro starter, I think we can flip back to `idle` once submission is complete. This is a good point though... we'll keep that in mind while revisiting `submitStatus`